backend/entrypoint_dev.sh

```
@@ -1,9 +1,14 @@
 #!/bin/sh
 
+# Requirements:
+# - Loads environment variables from .env file
+# - Starts TypeScript compiler in watch mode for development
+# - Uses ES modules for TypeScript compilation
+
 # Load environment variables from .env
 if [ -f .env ]; then
   export $(cat .env | sed 's/#.*//g' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)